tools/BugReportTool/BugReportTool/InstallationFolder.cpp

```diff
@@ -0,0 +1,216 @@
+#include "InstallationFolder.h"
+
+#include <fstream>
+#include <set>
+#include <Windows.h>
+#include <common/utils/winapi_error.h>
+
+using namespace std;
+using std::filesystem::directory_iterator;
```

Comment:
nit: btw we could do namespace aliases, e.g. `using fs = std::filesystem`

Reply:
I don't want to use acronyms for namespaces' names